src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        int numberR = random.nextInt(100);
+        String colorR = ColorSupplier.getRandomColor().toString();
+        return new Ball(colorR, numberR);
```

Comment:
That weird local var name.

I would propose to rename, e.g. `colorR` -> `randomColor` or `color`